frontend/src/metabase-types/types/Database.ts

```diff
@@ -22,12 +22,12 @@ export type DatabaseDetails = {
 
 export type DatabaseEngine = string;
 
-export type DatabaseNativePermission = "write" | "read";
+export type DatabaseNativePermission = "write" | "read" | "none";
 
 export type Database = {
   id: DatabaseId;
   name: string;
-  description?: string;
+  description: string | null;
```

Comment:
This type is deprecated, but I figured it won't hurt to fix the (incorrect) type.

Reply:
Should we maybe drop this type since you added types to the Database class?